mungegithub/mungers/blunderbuss.go

```diff
@@ -90,7 +93,11 @@ func printChance(owners weightMap, total int64) {
 	}
 }
 
-func getPotentialOwners(author string, feats *features.Features, files []*githubapi.CommitFile, leafOnly bool) (weightMap, int64) {
+type ownerVetter interface {
```

Comment:
You can get rid of this interface entirely and switch `getPotentialOwners` to use config directly.